web/modules/custom/usagov_wizard/usagov_wizard.module

```diff
@@ -92,6 +92,11 @@ function usagov_wizard_preprocess_html(&$variables) {
     $taxonomy_term = \Drupal::routeMatch()->getParameter('taxonomy_term');
     if (isset($taxonomy_term)) {
       if ($taxonomy_term->bundle() === 'wizard') {
+        $description = 'Just answer a few questions with USAGov\'s scam 
```

Comment:
You're going to need to conditionalize this on language and bring over the Spanish from the production wizard as well. It would be nice to collapse the white space in the description down too. (I don't know whether it actually matters to the tools that use it).

Reply:
Good point @akf. I went back to rework this and realized I didn't really do this the best way at first. I went ahead and set up a content based system for the metatag description, this will ensure that the value of the description in wizards can be managed through content. I updated the testing steps as well. Please retest when ready.